docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -904,3 +946,42 @@ To verify that the environment was removed, in your terminal window, run:
 
 The environments list that displays should not show the removed
 environment.
+
+.. _dry-run-explicit:
+
+Create explicit lockfiles without creating an environment
+=========================================================
+
+``@EXPLICIT`` lockfiles allow you to (re)create environments without invoking the solver.
+They consist of a ``@EXPLICIT`` header plus a list of conda package URLs optionally followed
```

Comment:
```suggestion
They consist of an ``@EXPLICIT`` header plus a list of conda package URLs, optionally followed
```
I'm not sure how ``@EXPLICIT`` is said, but I think it's either "at explicit" or "explicit", which both start with vowel sounds, so "an" is the correct article here.